src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CourtOfficerOrderHandler.java

```diff
@@ -75,6 +76,16 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse handleAboutToSubmit(CallbackParams callbackParams) {
+        var caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+
```

Comment:
Should this be behind a feature toggle? Otherwise It will go straight into production